lib/mozart_fetcher.ex

```diff
@@ -21,8 +21,23 @@ defmodule MozartFetcher do
     Application.get_env(:mozart_fetcher, :max_connections)
   end
 
-  def cert do
-    System.get_env("DEV_CERT_PEM")
+  def request_ssl do
+    case environment() do
+      :prod -> [certfile: cert_path(), cacertfile: ca_cert_path(), keyfile: key_file_path()]
+      _ -> [certfile: System.get_env("DEV_CERT_PEM")]
+    end
+  end
+
+  defp cert_path do
+    System.get_env("cert_file_path")
+  end
+
+  defp ca_cert_path do
+    System.get_env("ca_file_path")
+  end
+
+  defp key_file_path do
+    System.get_env("key_file_path")
```

Comment:
Are we keeping the same naming we use in requester, such as key_location?

Reply:
So I've intentionally not used requester's naming as I didn't think the naming for that was very good, as it switched between ending `_file` and `_location`. Now they all end in the same format, `_file_path`